docs/conmon.8.md

```diff
@@ -0,0 +1,140 @@
+conmon 8 "User Commands"
+==================================================
+
+# NAME
+
+conmon - container monitor utility
+
+# SYNOPSIS
+
+conmon [options]
+
+# DESCRIPTION
+
+conmon is a command line program for monitorying and managing the lifecycle of
+linux containers follow the Open Container Initiative (OCI) format.
+
+# APPLICATION OPTIONS
```

Comment:
Need periods at the end fo lines.  Many of the descriptions are basically a rehash of the option name and need more verbiage.

Reply:
These all come from the output of conmon --help.  So we really need to fix it in both places.